<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickcharsDialog</class>
 <widget class="QDialog" name="PickcharsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeePassXC - Pick Characters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Select characters to type, navigate with arrow keys, Ctrl + S submits.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="charsGrid"/>
   </item>
   <item>
    <widget class="PasswordWidget" name="selectedChars">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="pressTab">
       <property name="text">
        <string>Press &amp;Tab between characters</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordWidget</class>
   <extends>QLineEdit</extends>
   <header>gui/PasswordWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>selectedChars</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
